base_layer/wallet/src/output_manager_service/protocols/utxo_validation_protocol.rs

```diff
@@ -135,6 +135,18 @@ where TBackend: OutputManagerBackend + Clone + 'static
                 target: LOG_TARGET,
                 "UTXO validation protocol (Id: {}) has no outputs to validate", self.id,
             );
+            let _ = self
```

Comment:
I'm not 100% sure about swallowing the result here, but I see that's how it's done in other places in the file.